codegenerator/cli/templates/static/codegen/src/Index.bs.js

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node
```

Comment:
Should look like this, but for some reason it's not included to the generated directory

<img width="710" alt="image" src="https://github.com/user-attachments/assets/67fec9d4-8acf-4400-91c2-bbd96657c037" />

Reply:
Ok, looks like it's working correctly now